src/LibraryViewExtensionWebView2/Handlers/SearchResultDataProvider.cs

```diff
@@ -48,7 +49,7 @@ public SearchResultDataProvider(NodeSearchModel model, IconResourceProvider icon
         public override Stream GetResource(string searchText, out string extension)
         {
             var text = Uri.UnescapeDataString(searchText);
-            var elements = model.Search(text, 0, null);
+            var elements = model.Search(text, dynamoModel.LuceneSearchUtility);
```

Comment:
Hmm looks like we actually want @reddyashish 's change so we can access the search utility without DynamoModel